src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerFileReference.cs

```diff
@@ -70,6 +75,12 @@ public override ImmutableArray<DiagnosticAnalyzer> GetAnalyzers(string language)
             return _diagnosticAnalyzers.GetExtensions(language);
         }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("ApiDesign", "RS0016:Add public types and members to the declared API", Justification = "In Progress")]
+        public override ImmutableArray<ISourceGenerator> GetGenerators()
```

Comment:
>GetGenerators [](start = 57, length = 13)

Are there tests for this method?

Reply:
Added

---
In reply to: [389827191](https://github.com/dotnet/roslyn/pull/42165#discussion_r389827191) [](ancestors = 389827191)